### DISCUSSION TITLE
Release policy

### DISCUSSION THREAD

**Discussion by PhilippDehner** [ActivityWatch/activitywatch#1011] [General]
Hi, 

how often will releases be published? There are many new commits, but almost one year no new releases.
Thanks!

**Comment by ErikBjare** [comment]
We do not have a fixed release schedule, although I try to release at least one major version each year. But as you've noticed, the one year mark is coming up soon...

The reason is that I'm building this in my spare time, and want to make sure any stable release is stable enough that I won't have to hurry out any fixes. Therefore, creating a stable release usually takes more time than I'd like, as we have to do testing across 3 operating systems.

We do however make beta releases sporadically, which you can find on the [Releases](https://github.com/ActivityWatch/activitywatch/releases) page. However, they might not always work though (the latest `v0.12.3b16` is broken on macOS, for example).

**Reply by PhilippDehner** [reply]
@ErikBjare Ahh okay, thank you very much for the answer!
